hcl2template/types.packer_config.go

```diff
@@ -150,10 +150,18 @@ func (c *PackerConfig) parseLocalVariables(f *hcl.File) ([]*LocalBlock, hcl.Diag
 
 	content, moreDiags := f.Body.Content(configSchema)
 	diags = append(diags, moreDiags...)
-	var locals []*LocalBlock
+
+	var locals = c.LocalBlocks
+	if locals == nil {
+		locals = []*LocalBlock{}
+	}
```

Comment:
This check is only necessary for maps, the code should work without this.
```suggestion
	locals := c.LocalBlocks
```